src/encode.ts

```diff
@@ -178,6 +179,13 @@ type Options = {
 	 * @default true
 	 */
 	deburr: boolean;
+
+	/**
+	 * If true, validates data model before encoding it.
+	 *
+	 * @default false
+	 */
+	validateModel: boolean;
```

Comment:
I initially suggested a default of false, but after thinking it over, I believe true would be more expected

Reply:
@xseman - this will cause validation to run by default in the terminal without an option to turn it out. Should I add an optional flag `withoutValidation` to the cli command?

Reply:
[c1bf714](https://github.com/xseman/bysquare/pull/43/commits/c1bf7148298c60d1de8caae4b91164d97320a31e)

Reply:
![image](https://github.com/user-attachments/assets/f30c1008-a1ab-4d5d-9be9-31b5bd3f1281)

![image](https://github.com/user-attachments/assets/e1abd578-1fbe-4b1a-b041-f7cf5d1a9b9e)

[d27658e](https://github.com/xseman/bysquare/pull/43/commits/d27658ea9bb983aff6793284f9e9d2c43dafb739)